backend/private-graph/graph/middleware.go

```diff
@@ -128,13 +128,13 @@ func authenticateToken(tokenString string) (jwt.MapClaims, error) {
 		return claims, e.Wrap(err, "invalid id token")
 	}
 
-	exp, ok := claims["exp"].(float64)
+	exp, ok := claims["exp"]
 	if !ok {
 		return claims, e.Wrap(err, "invalid exp claim")
 	}
 
-	// Check if the current time is after the expiration
-	if time.Now().After(time.Unix(int64(exp), 0)) {
+	expClaim := int64(exp.(float64))
```

Comment:
ah, so the claim exp is given as a `float64` by the jwt library... interesting. my bad confusing you here, thanks for fixing this!

Reply:
Yeah. It also escaped me.